frappe/desk/doctype/auto_repeat/auto_repeat.py

```diff
@@ -354,13 +367,13 @@ def auto_repeat_doctype_query(doctype, txt, searchfield, start, page_len, filter
 			if(locate(%(_txt)s, parent), locate(%(_txt)s, parent), 99999),
 			parent
 		limit %(start)s, %(page_len)s""".format(**{
-			'key': searchfield,
-		}), {
-			'txt': "%%%s%%" % txt,
-			'_txt': txt.replace("%", ""),
-			'start': start,
-			'page_len': page_len
-		})
+		'key': searchfield,
+	}), {
+							 'txt': "%%%s%%" % txt,
```

Comment:
please use only 1 indent, not multiple

Reply:
Okay! I will fix the conflict.

Reply:
@iRaySpace reminder :)

Reply:
Hello @rmehta Working on rebase. Thank you for reminding.

Reply:
Hello @rmehta I think I should re-do my customization by new branch based from the latest develop and open a new pull request. It seems my rebase is done wrong.